### PR TITLE
[CNTT TSC] Baraque Release Details

### LINKED ISSUE #1703: [TSC] Record Baraque release features.
After Issue logging is complete, a PR is needed to log in baroque release featreuset.

### PR DESCRIPTION
**Pull request by rabi-abdel**
Fixes #1703

### CODE REVIEW

File: doc/ref_arch/kubernetes/README.md [hunk 1]
```diff
@@ -5,18 +5,19 @@
...
-**Release Date: _15th May 2020_**
+**Release Date: _25th September 2020_**
 
 ## Bundle/Version History
 
 | Bundle.Version | Date | Note
 | --- | --- | --- |
 | 1.0-alpha | 10th January 2020 | Snezka Release|
 | 3.0 | 15th May 2020 | Baldy Release|
+| 4.0 | 25th Sep 2020 | Baraque Release|
```
Comment (collivier) [inline doc/ref_arch/kubernetes/README.md:20]: I would prefer us to format the tabs when easily doable

File: doc/ref_arch/openstack/README.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ This is Openstack based Reference Architecture (RA-1)
 | 1.0 | 20th September 2019 | Botrange Release|
 | 2.0-alpha | 10th January 2020 | Snezka Release|
 | 2.0 | 15th May 2020 | Baldy Release|
+| 3.0| 25th Sep 2020 | Baraque Release|
```
Comment (collivier) [inline doc/ref_arch/openstack/README.md:21]: ditto

Comment (rabi-abdel) [reply doc/ref_arch/openstack/README.md]: fixed in https://github.com/cntt-n/CNTT/pull/1739/commits/02d57baad26960976fd5e17ff9949b5a8d119cd7

File: doc/ref_arch/kubernetes/README.md (hunk 1 above)
Comment (rabi-abdel) [reply doc/ref_arch/kubernetes/README.md]: fixed in https://github.com/cntt-n/CNTT/pull/1739/commits/02d57baad26960976fd5e17ff9949b5a8d119cd7

File: doc/ref_arch/kubernetes/README.md [hunk 3]
```diff
@@ -5,18 +5,19 @@
...
 ## Bundle/Version History
 
-| Bundle.Version | Date | Note
-| --- | --- | --- |
-| 1.0-alpha | 10th January 2020 | Snezka Release|
-| 3.0 | 15th May 2020 | Baldy Release|
+| Bundle.Version    | Date              | Note
+| ---               | ---               | ---               |
+| 1.0-alpha         | 10th January 2020 | Snezka Release    |
+| 3.0               | 15th May 2020     | Baldy Release     |
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/README.md:19]: why did we jump a release for RA-2? If we make this v2.0 for Baldy, Baraque will only be v3.0 aligned with others.

Comment (collivier) [reply doc/ref_arch/kubernetes/README.md:19]: +1

Comment (rabi-abdel) [reply doc/ref_arch/kubernetes/README.md:19]: we discussed this before though the bundle discussion. for RA-2, we are aligning it with RM latest release.

**Comment by rabi-abdel** [comment]
@collivier can you please re-review

**Comment by collivier** [comment]
@rabi-abdel @rabi-abdel I may be wrong but it seems that OVP Badge is still an RC1 objective.
I could fix all tabs if it helps.

I would have proposed as an RA1 and RC1 objectives to verify if specific parts could be hosted by GSMA.
The OpenStack API requirements, the test case principles, descriptions, and selections could be considered at first glance.
Hosting our outcomes in GSMA is also our target. Our CNT CNTT objectives would be at least to discuss the feasability and more if possible.

**Comment by collivier** [comment]
Maybe the issue is related to the force push.

**Comment by rabi-abdel** [comment]
> Maybe the issue is related to the force push.

@collivier would you mind pointing out to the exact place where OVP is mentioned and you would like to recommend a change? also it might help if you propose commit changes would would simplify it.

**Comment by rabi-abdel** [comment]
@collivier can you please re-review based on latest update

**Comment by rabi-abdel** [comment]
48 hours passed since last feedback. approving and merging.

**Pull request merged**